contracts/cw721-base/src/msg.rs

```diff
@@ -1,179 +1,5 @@
-use cosmwasm_schema::{cw_serde, QueryResponses};
-use cosmwasm_std::{Binary, Coin};
-use cw721::Expiration;
-use cw_ownable::{cw_ownable_execute, cw_ownable_query};
-use schemars::JsonSchema;
-
-#[cw_serde]
-pub struct InstantiateMsg {
```

Comment:
All msgs and response structs moved from `cw721-base` to `cw721` for better re-use